"panic: runtime error: slice bounds out of range" while "oc adm release extract"

**Describe the bug**
Just running a `make` within the https://github.com/openshift-metal3/dev-scripts/ with my usual hardware/environment 

**To Reproduce**
From dev-scripts  b37f69396 commit, RHEL8 helper node

**Expected/observed behavior**
```
+ oc adm release extract --registry-config installer--q65IkgS0h2/pullsecret --command=openshift-install --to installer--q65IkgS0h2 registry.svc.ci.openshift.org/kni/release:4.2.0-0.ci-2019-07-31-123929-kni.0
panic: runtime error: slice bounds out of range

goroutine 163 [running]:
bufio.(*Reader).fill(0xc000f35c80)
        /opt/rh/go-toolset-1.12/root/usr/lib/go-toolset-1.12-golang/src/bufio/bufio.go:89 +0x211
bufio.(*Reader).WriteTo(0xc000f35c80, 0x2dfa380, 0xc000010838, 0x7f60a71fde08, 0xc000f35c80, 0x1)
        /opt/rh/go-toolset-1.12/root/usr/lib/go-toolset-1.12-golang/src/bufio/bufio.go:511 +0x106
io.copyBuffer(0x2dfa380, 0xc000010838, 0x2df4f60, 0xc000f35c80, 0x0, 0x0, 0x0, 0xc0013e5260, 0xc0000edb00, 0x0)
        /opt/rh/go-toolset-1.12/root/usr/lib/go-toolset-1.12-golang/src/io/io.go:384 +0x34e
io.Copy(...)
        /opt/rh/go-toolset-1.12/root/usr/lib/go-toolset-1.12-golang/src/io/io.go:364
os/exec.(*Cmd).stdin.func1(0xc000ef4900, 0x0)
        /opt/rh/go-toolset-1.12/root/usr/lib/go-toolset-1.12-golang/src/os/exec/exec.go:243 +0x67
os/exec.(*Cmd).Start.func1(0xc0008f34a0, 0xc0014f7680)
        /opt/rh/go-toolset-1.12/root/usr/lib/go-toolset-1.12-golang/src/os/exec/exec.go:409 +0x27
created by os/exec.(*Cmd).Start
        /opt/rh/go-toolset-1.12/root/usr/lib/go-toolset-1.12-golang/src/os/exec/exec.go:408 +0x58f
make: *** [Makefile:21: ocp_run] Error 2
```

**Additional context**
Trying again seems to work and I've not been able to reproduce it.

Reported first here https://github.com/openshift-metal3/dev-scripts/issues/734

--- a/pkg/cli/image/extract/extract.go
+++ b/pkg/cli/image/extract/extract.go
@@ -483,6 +483,10 @@ func (o *Options) Run() error {
 }
 
 func layerByEntry(r io.Reader, options *archive.TarOptions, layerInfo LayerInfo, fn TarEntryFunc, allLayers bool, alreadySeen map[string]struct{}) (bool, error) {
+	// Prevents race condition present in vendored version of docker
+	// https://github.com/moby/moby/issues/39859
+	os.Setenv("MOBY_DISABLE_PIGZ", "true")
+
 	rc, err := dockerarchive.DecompressStream(r)
 	if err != nil {
 		return false, err

